xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/internal/reference/SymbolScheme.java

```diff
@@ -94,6 +104,17 @@
      */
     String[] getParameterReplacementSymbols(EntityType type);
 
+    /**
+     * @param parameter the name of the parameter
+     * @param value the String value of the parameter
+     * @return the parameter converted in the expected type
```

Comment:
WDYT of adding examples of expected values for the parameters? I feel like it could help understanding the method.

Reply:
Yeah, I guess I could give the Locale example, which is the only use case currently.